src/module.cc

```diff
@@ -68,6 +69,21 @@ NAN_METHOD(TriggerReport) {
 }
 
 /*******************************************************************************
+ * External JavaScript API for returning a NodeReport
+ *
+ ******************************************************************************/
+NAN_METHOD(GetReport) {
+  Nan::HandleScope scope;
+  v8::Isolate* isolate = info.GetIsolate();
+  std::ostringstream out;
+
+  GetNodeReport(isolate, kJavaScript, "JavaScript API", __func__, out);
+  // Return value is the NodeReport filename
```

Comment:
This comment looks wrong, probably cut&pasted, the return value for this API is the report string